common/app/redux/fetch-youtube-saga.js

```diff
@@ -0,0 +1,129 @@
+import { Observable } from 'rx';
+
+import { get$ } from '../../utils/ajax-stream';
+import { dashify } from '../../utils';
+import types from './types';
+import {
+  createErrorObservable,
+  updateBlock,
+  updateSuperBlock,
+  updateYoutube
+} from './actions';
+import {
+  initMap
+} from '../routes/challenges/redux/actions';
+import {
+  createMapUi,
+  searchableChallengeTitles
+} from '../routes/challenges/utils';
+
+const { fetchYoutube } = types;
+export default function fetchYoutubeSaga(action$, getState) {
+  return action$
+    .filter(action => action.type === fetchYoutube)
+    .flatMap(() => {
+      const url = 'https://glitter-organisation.gomix.me/api/v1/youtube';
```

Comment:
Should this API handle traffic from all freeCodeCamp's users? I assume it's cached for a while, but would Gomix be happy to host this API? :stuck_out_tongue: